app/main/posts/detail/detail.html

```diff
@@ -81,10 +81,20 @@ <h2 class="form-label" translate>post.modify.form.categories</h2>
             {{category.tag}}
           </p>
         </div>
+        <div
+          ng-repeat="(key,value) in post.values"
```

Comment:
This leaves us looping through `post.values` twice, and potentially showing thing out of order. We should at least reduce this to a single ng-repeat above both post-value and post-video-value.. or as suggested below move post-video-value inside post-value